satpy/scene.py

```diff
@@ -233,7 +234,20 @@ def max_area(self, datasets=None):
         """
         return self._compare_areas(datasets=datasets, compare_func=max)
 
-    def min_area(self, datasets=None):
+    def max_area(self, datasets=None):
+        """Get highest resolution area for the provided datasets. Deprecated.
+
+        Args:
+            datasets (iterable): Datasets whose areas will be compared. Can
+                                 be either `xarray.DataArray` objects or
+                                 identifiers to get the DataArrays from the
+                                 current Scene. Defaults to all datasets.
+
+        """
+        warnings.warn("'max_area' is deprecated, use 'finest_area' instead.")
```

Comment:
I think @gerritholl wants these types of warnings to be `DeprecationWarning` now instead of the default `UserWarning`.